src/Application/Authorisation/Suggest/View.elm

```diff
@@ -84,6 +89,16 @@ view context model =
                 , Html.text "?"
                 ]
 
+          else if hasRawResources then
```

Comment:
This prompt does not include a timespan. The user must determine expiration time from the displayed JSON.